beancount-tests.el

```diff
@@ -348,3 +351,95 @@ known option nmaes."
       (goto-char 0)
       (beancount-date-down-day)
       (should (equal (thing-at-point 'line) "2024-05-10\n"))))
+
+;;; Xref backend
+
+(defun beancount-test-xref-definition-pos (identifier position)
+  (let ((defs (xref-backend-definitions 'beancount identifier)))
```

Comment:
Can you please add a comment that says what this function does? Not being familiar with the xref implementation, it is not immediately obvious to me